### ISSUE TITLE
Incorrect Widget Display

### ISSUE THREAD

**Issue by madzman23** [Eonasdan/bootstrap-datetimepicker#2165]
Hello, I have this problem on the widget after I changed the value of the date, then refresh the page. When I click the datetimepicker icon, the widget didn't displayed the correct value of the date. But I think the focus is in the correct date. After I clicked the icon again, I get the correct value.

See this image: https://drive.google.com/open?id=0ByQNz5VQBB2PZS1JazZQVnlkZ0k

I tried to fix it by forcing the date field value to the correct date, please see this [fiddle](http://jsfiddle.net/madzman23/kspetgr2/4/). But it's still not working. Can anyone help me, sort this out? Thanks.

**Comment by William-H-M** [comment]
HI, I see that on the fiddle that you provided you're using V. 4.14.30, could you use latest version and see what happens? if still going can you provide something like a gif showing what happend as on your fiddle I'm unable to reproduce the error

**Comment by madzman23** [comment]
Shoot. I didn't checked the versions. My bad. The fiddle I'm using is v4.14.30, it works with that version and I also cannot replicate the error. The version I used on my application is v4.17.43, I think that is the bug in that version. I tested using the version 4.14.30 and then tested it with v4.17.47, it all worked well. So I already updated it with the latest version 4.17.47. Thank you very much @William-H-M! Cheers..
